hocr-check

```diff
@@ -80,26 +88,26 @@ doc = html.fromstring(stream.read())
 ################################################################
 
 # check for presence of meta information
-assert doc.xpath("//meta[@name='ocr-id']")!=[]
-assert doc.xpath("//meta[@name='ocr-recognized']")!=[]
+test_ok(doc.xpath("//meta[@name='ocr-id']")!=[], "//meta[@name='ocr-id']")
+test_ok(doc.xpath("//meta[@name='ocr-recognized']")!=[], "//meta[@name='ocr-recognized']")
 
 # check for presence of page
-assert doc.xpath("//*[@class='ocr_page']")!=[]
+test_ok(doc.xpath("//*[@class='ocr_page']")!=[], "has a page")
 
 # check that lines are inside pages
 lines = doc.xpath("//*[@class='ocr_line']")
-for line in lines:
-    assert line.xpath("//*[@class='ocr_page']")
+for line_idx, line in enumerate(lines):
+    test_ok(line.xpath("//*[@class='ocr_page']"), "ocr_line %2d in a ocr_page"%(line_idx))
```

Comment:
They are supposed to check for the opposite but don't and should be fixed. All those tests in for loops.